tensorflow/contrib/ios_examples/README.md

```diff
@@ -13,7 +13,17 @@ This folder contains examples of how to build applications for iOS devices using
  - Download
    [Inception v1](https://storage.googleapis.com/download.tensorflow.org/models/inception5h.zip),
    and extract the label and graph files into the data folders inside both the
-   simple and camera examples.
+   simple and camera examples:
+
+```bash
+mkdir -p ~/graphs
+curl -o ~/graphs/inception.zip \
+ https://storage.googleapis.com/download.tensorflow.org/models/inception5h.zip \
+ && unzip ~/graphs/inception.zip -d ~/graphs/inception
```

Comment:
Thanks, I've updated all of the directions to refer to inception5h rather than just inception.